features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/Create/CreateBusinessPlan.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ import React, {Component} from 'react'
+
+ import {Link} from 'react-router-dom'
+ import AppBar from 'material-ui/AppBar';
+ import Toolbar from 'material-ui/Toolbar';;
+ import IconButton from 'material-ui/IconButton';
+ import Button from 'material-ui/Button';
+ import MenuIcon from 'material-ui-icons/Menu';
+
+ import PropTypes from 'prop-types';
+ import { withStyles } from 'material-ui/styles';
+ import TextField from 'material-ui/TextField';
+ import Typography from 'material-ui/Typography';
+ import Divider from 'material-ui/Divider';
+ import Grid from 'material-ui/Grid';
+ import { FormControl, FormLabel, FormControlLabel } from 'material-ui/Form';
+ import Radio, { RadioGroup } from 'material-ui/Radio';
+ import Paper from 'material-ui/Paper';
+
+ class CreateBusinessPlan extends Component{
+        render() {
+           return (
+            <div>
+                 <AppBar position="static" >
+                     <Toolbar style={{minHeight:'30px'}}>
+                         <IconButton color="contrast" aria-label="Menu">
+                             <MenuIcon />
+                         </IconButton>
+                         <Link to={"/policies/business_plans"}>
+                              <Button color="contrast">Go Back</Button>
```

Comment:
shouldn't this be labelled as "back" in oppose to "go back" ?

Reply:
Is there any specific reason for that?

Reply:
IMO it's better to use "Back" as it implies "go back". but no particular reason. it would also make the button longer isnt it?